walletrpc/compact_formats.proto

```diff
@@ -55,24 +55,56 @@ message CompactTx {
     repeated CompactSaplingSpend spends = 4;
     repeated CompactSaplingOutput outputs = 5;
     repeated CompactOrchardAction actions = 6;
+    // CompactTxIn values  corresponding to the `vin` entries of the full transaction.
+    //
+    // Note: coinbase transactions will have no entries in `vin`.  Light Clients can
+    // test `CompactTx.index == 0` to determine whether a `CompactTx` represents a
+    // it's a coinbase transaction, as the coinbase transaction is always the first transaction
+    // in any block.
+    repeated CompactTxIn vin = 7;
+    repeated TxOut vout = 8;
+}
+
+// The unique identifier for a transparent transaction output.
+message OutPoint {
```

Comment:
Do we need a separate `OutPoint` message type, or can we just embed these fields into `CompactTxIn`? What else would this message be used for?